package.json

```diff
@@ -5,23 +5,24 @@
   "homepage": "https://pokejalasoft.github.io/PokeJalasoftProject/",
   "type": "module",
   "scripts": {
-    "predeploy" : "npm run build",
-    "deploy" : "gh-pages -d dist",
+    "predeploy": "npm run build",
```

Comment:
Restore spaces

Reply:
Actually not delete it there are top down